bigtable/src/test/java/com/m/examples/bigtable/QuickstartTest.java

```diff
@@ -56,6 +58,11 @@ public void setUp() {
     System.setOut(new PrintStream(bout));
   }
 
+  @After
+  public void tearDown() {
+    System.setOut(oldStdout);
```

Comment:
```suggestion
    System.setOut(null);
    bout.reset();
```
I am more use to seeing something like this.